### ISSUE TITLE
recursive clone error

### ISSUE THREAD

**Issue by Solomon1588** [myfavouritekk/TPN#1]
I cloned the repository recursively, but I got a error during cloning the vdetlib and caffe-mpi
My git script:
`git clone --recursive https://github.com/myfavouritekk/TPN.git`
@myfavouritekk What can I do to clone the entire repository? THX

```
fatal: early EOFs:  13% (3934/29114), 4.39 MiB | 8.00 KiB/s
fatal: The remote end hung up unexpectedly
fatal: index-pack failed
error: RPC failed; curl 18 transfer closed with outstanding read data remaining
fatal: clone of 'https://github.com/myfavouritekk/caffe.git' into submodule path 'E:/workSpace/models/TPN/external/caffe-mpi' failed
Failed to clone 'external/caffe-mpi'. Retry scheduled

Cloning into 'E:/workSpace/models/TPN/external/vdetlib'...
ssh_exchange_identification: read: Connection reset by peer
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
fatal: clone of 'git@github.com:myfavouritekk/vdetlib.git' into submodule path 'E:/workSpace/models/TPN/external/vdetlib' failed
Failed to clone 'external/vdetlib'. Retry scheduled
Cloning into 'E:/workSpace/models/TPN/external/caffe-mpi'...
remote: Counting objects: 29114, done.
remote: Compressing objects: 100% (3/3), done.
warning: Clone succeeded, but checkout failed. | 14.00 KiB/s
You can inspect what was checked out with 'git status'
and retry the checkout with 'git checkout -f HEAD

```

**Comment by Solomon1588** [comment]
And now it's all OK :D

**Comment by myfavouritekk** [comment]
@Solomon1588 Fixed in f078927f8d01fcd190af98c611d19aeeee6da4e1
